helper/schema/valuetype_string.go

```diff
@@ -9,7 +9,7 @@ const _ValueType_name = "TypeInvalidTypeBoolTypeIntTypeFloatTypeStringTypeListTy
 var _ValueType_index = [...]uint8{0, 11, 19, 26, 35, 45, 53, 60, 67, 77}
 
 func (i ValueType) String() string {
-	if i < 0 || i >= ValueType(len(_ValueType_index)-1) {
+	if i < 0 || i+1 >= ValueType(len(_ValueType_index)) {
```

Comment:
Yes it accidentally sneaked in, my bad! Will revert it in a few (if not done already). This is caused by doing these tweaks on an out of date macbook (so with an older stringer package)...